uint32 timestamp_ms		# in milliseconds since system start
int8[10] key			# max. 10 characters as key / name
float32 value			# the value to send as debug output
